app/controllers/repos_controller.rb

```diff
@@ -92,33 +93,32 @@ def show_releases
           benchmark_runs << latest_benchmark_run
         end
 
-        columns = ChartBuilder.new(benchmark_runs).build_columns do |benchmark_run|
+        chart_builder = ChartBuilder.new(benchmark_runs, benchmark_result_type)
+        chart_builder.build_columns do |benchmark_run|
           environment = YAML.load(benchmark_run.environment)
 
-          if environment.is_a?(Hash)
-            temp = ""
-
-            environment.each do |key, value|
-              temp << "#{key}: #{value}<br>"
-            end
+          version = { version: benchmark_run.initiator.version }
 
-            environment = temp
+          # If there is more information about the environment, we add it to `version` 
+          if environment.is_a?(Hash)
```

Comment:
we use the resulting `version` hash to generate our json as well as the chart popups: 
![image](https://cloud.githubusercontent.com/assets/15824429/23272976/53c5e558-f9cb-11e6-96c3-b6267bc47fbe.png)
so those could end up being inconsistent

`environment` is just extra info that we add on, the important stuff (commit sha, commit date, release version, etc.) will always be consistent